tools/x-repo-types/xrt.go

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2019-2023 Algorand, Inc.
```

Comment:
do you still need this file now that you have typeAnalyzer?

Reply:
Yes. Perhaps this question suggest that I made a poor naming choice for the `runner/` directory. `xrt` is the binary created for `xrt.go` and it compiles and runs the objects in `runner/`. Do you thing I should rename the directory to `typeAnalyzer/` or something else?

Reply:
Resolving. The naming discussion is below.